File: 070-爬楼梯/070.java
class Solution {
    public int climbStairs(int n) {
        if(n==1){return 1;}
        int[] res = new int[n];
        res[0] = 1;
        res[1] = 2;
        for(int i=2;i<n;i++){
            res[i] = res[i-1] + res[i-2];
        }
        return res[n-1];
    }
}

File: 070-爬楼梯/070.cpp
class Solution {
public:
	// c++有时候会报错 很迷 是数组开的太大了吗
    int climbStairs(int n) {
        vector<int> res;
        res.push_back(0);
        res.push_back(1);
        res.push_back(2);
        for(int i =3;i<=n;i++){
            res.push_back(res[i-1] + res[i-2]);
        }
        return res[n];
    }
};